### ISSUE TITLE
Replace images on flippy cards with broken images

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/pomegranate#711]
This is the case for some unpublished exhibits.

Or, if the exhibits are no longer in development, delete them. related to #709 

This was linked to a HB issue but on further inspection that issue appears unrelated.

**Comment by kelea99** [comment]
@hackmastera , I cannot delete this collection, as it says it doesn't exist. can you help? thanks!  https://dpul.princeton.edu/dpul.princeton.edu%2Fslavic

**Comment by kelea99** [comment]
@hackmastera, I went in to dashboard for Gutenberg and After and just re-saved the flippy card selection.  I checked with Firefox, chrome and safari again and it is working for all three.

**Comment by hackmastera** [comment]
@kelea99 Thank you! It turned out the Gutenberg exhibit was broken due to a lost mount point. It was working sometimes because one of our web servers had the mount, but the other did not.

Sorry for confusion I caused in editing this issue. I started repurposing it and then ended up making a new issue (#767) instead. fixing the mount point does appear to have resolved any broken images we had, so I'll close this one also.
